diff --git a/CMakeLists.txt b/CMakeLists.txt
index dae7bedf..13876498 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -522,7 +522,6 @@ if (CLING_BUILT_STANDALONE OR CMAKE_VERSION VERSION_EQUAL 3 OR
     ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/ClingConfig.cmake
     DESTINATION lib${LLVM_LIBDIR_SUFFIX}/cmake/cling)
 
-  install(DIRECTORY tools/Jupyter/kernel DESTINATION share/cling/Jupyter)
 
 
   # Also copy ClingConfig.cmake to the build directory so that dependent projects
diff --git a/tools/CMakeLists.txt b/tools/CMakeLists.txt
index 67881875..49c436bf 100644
--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -9,7 +9,6 @@
 if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/../lib/UserInterface/textinput
    OR CLING_INCLUDE_TESTS)
   add_subdirectory(driver)
-  add_subdirectory(Jupyter)
   add_subdirectory(libcling)
   add_subdirectory(demo)
 endif()
